rfc6962-bis.xml

```diff
@@ -656,7 +655,7 @@ updated on the fly.
 (see <xref target="cert_sctlist_extension"/>). This mechanism allows the use of
 unmodified TLS servers, but, because the included SCTs cannot be changed without
 re-issuing the certificate, increases the risk that the certificate will be
-refused if any of the SCTs become invalid.
+considered non-compliant if some of the SCTs become invalid (by the client not recognizing the logs they were issued from anymore).
```

Comment:
Note that my version also removed the notion of SCTs "becoming invalid".  The signature on an SCT doesn't cease to be valid just because the log that signed the SCT is no longer accepted by the client.